### DISCUSSION TITLE
RandomLinkSplit Load_csv tutorial on hetero

### DISCUSSION THREAD

**Discussion by therealadarsh** [pyg-team/pytorch_geometric#5996] [Q&A]
Hi, 
I am trying to create a new graph dataset using hetero tutorial from load_csv. I was able to add the data and sorta follow up well until the RandomLinkSplit. I keep getting this error which basically the attribute edge_index is not present. But I can see it when I print data. Any tips on what I am missing? I have added the screenshot for your reference.
![err](https://user-images.githubusercontent.com/54485879/202364697-ada6aa8e-b25b-494d-ac10-b40c974753f7.jpg)

**Comment by rusty1s** [comment]
That is a bit strange. Just from looking at the images, that should definitely work. Do you have an example to reproduce on our end? Does
```python
for edge_type in data.edge_types:
    print(edge_type)
    print(data[edge_type].edge_index)
```
works for you?
